doc/rtd/development/integration_tests.rst

```diff
@@ -65,6 +204,39 @@ For a minimal setup using LXD, write the following to
 
     [lxd]
 
+
+For more information on configuring pycloudlib, see the
+`pycloudlib configuration documentation`_.
+
+To specify a specific cloud to test against, first, ensure that your pycloudlib
+configuration is set up correctly. Then, modify the ``PLATFORM`` variable to be
+on of:
+
+- ``azure``: Microsoft Azure
+- ``ec2``: Amazon EC2t
```

Comment:
Whoops. After this one we should be really actually for sure good :wink:

Reply:
haha great catch. thanks @TheRealFalcon !